<launch>
  <node name="stereo_dvs_node" pkg="dvs_stereo" type="dvs_stereo" output="screen">

  <remap from="/left_events" to="/event_out_left/slave"/>
  <remap from="/right_events" to="/event_out_right/master"/>

  <param name="camera_height" type="int" value="720" />
  <param name="camera_width" type="int" value="1280" />

  <param name="age_penalty" type="double" value="0.0001" />
  <param name="msg_queue_size" type="int" value="100" />

  <param name="write_to_text" type="bool" value="false" />
  <param name="calc_disparity" type="bool" value="true" />
  <param name="do_adaptive_window" type="bool" value="false" />
  <param name="nearest_neighbour" type="bool" value="false" />
  <param name="rectify" type="bool" value="false" />
  <param name="checker_board_window" type="bool" value="false" />

  <param name="disparity_range" type="int" value="128" />
  <param name="disparity_step" type="int" value="3" />

  <param name="step_size" type="int" value="3" />
  <param name="window_fill_size" type="int" value="2" />

  <param name="NN_min_num_of_events" type="int" value="5" />
  <param name="NN_block_size" type="int" value="10" />

  <param name="small_block_size" type="int" value="5" />
  <param name="medium_block_size" type="int" value="11" />
  <param name="large_block_size" type="int" value="13" />

  </node>

</launch>
